.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,46 @@
+name: "Close stale issues"
```

Comment:
We can’t auto close stale issues as crt-swift uses issues for planning/tracking purposes.

Reply:
commented out the part that will close issues at one year. Left in the bot that closes issues after response requested